.eslintrc.js

```diff
@@ -44,10 +44,19 @@ module.exports = {
         "indent": ["warn", 4, {"SwitchCase":1}],
         "linebreak-style": ["error", "unix"],
         "semi": ["error", "always"],
+        "keyword-spacing": ["error"],
+        "no-implicit-globals": ["error"],
+        "no-irregular-whitespace": ["error"],
         "no-new-object": ["error"],
-        "no-unneeded-ternary": ["error"],
+        "no-regex-spaces": ["error"],
+        "no-spaced-func": ["error"],
```

Comment:
Looks like this has been deprecated in favor of
```
"func-call-spacing": ["error", "never"]
```
according to https://eslint.org/docs/rules/no-spaced-func#disallow-spacing-between-function-identifiers-and-their-applications-no-spaced-func. (Though what's here still works on the current version and does the same thing.)